images/fcos-bastion-image/root/usr/bin/clean-registries.sh

```diff
@@ -21,15 +21,13 @@ wait_for_container_running() {
 mapfile -t ports < <(systemctl list-units 'registry@*' --no-pager --quiet | awk -F'[@.]' '{print $2}')
 for port in "${ports[@]}"; do
   disk_use=$(df /opt/registry-"${port}" --output='pcent' | grep -o '[0-9]*')
-  if [ "$disk_use" -gt 85 ]; then
+  if [ "$disk_use" -gt 60 ]; then
     rm -rf /opt/registry-"${port}"/data/docker/registry/v2/repositories/*
-    sed -i -e '/readonly/{n;s/enabled:.*/enabled: true/;}' /opt/registry-"${port}"/config.yaml
-    systemctl restart registry@"${port}".service
-    wait_for_container_running registry-"${port}"
+  fi
```

Comment:
i see you're removing the readonly. Is the garbage-collect command concurrent-safe?

Reply:
No, I think not... Sorry, I mistook it to be "delete: enabled". But we didn't have 'readonly' keyword in config.yaml file, does it really working? Let me add it to gen_registry_file.sh.